subprojects/testing-native/src/integTest/groovy/org/gradle/nativeplatform/test/xctest/XCTestIntegrationTest.groovy

```diff
@@ -90,6 +94,27 @@ apply plugin: 'xctest'
         testBundle.assertTestCasesRan(testExecutionResult)
     }
 
+    def 'can build xctest bundle which depends multiple swift modules'() {
```

Comment:
I agree we should look at making what we have better. Spending some time improving them would be more beneficial. It is hard to understand the fixtures as they are but I think it's just a matter of better structuring them (and possibly naming). I'm not against the idea of `TestResources`. We can talk about it tomorrow.

Reply:
I'm not sure how to proceed here concretely. The current set of test fixtures feel very complected and entangled. I like the idea of being able to see what the project looks like without having to create a failing test. That's why I like the simplicity of the `Resources` based test infrastructure.  It is nice to be able to say, `assertTestCasesRun` but, I'm wondering if we could create something similar to that which just parses the sources and the test reports and makes sure they match? But, also, I'm not sure how useful it actually is. I guess we are concerned that a bug in the testing infrastructure would cause some test cases to be ignored and the test task would still be successful? I'm open to ideas here.